### ISSUE TITLE
Can't install powershell_es

### ISSUE THREAD

**Issue by sakhnik** [williamboman/nvim-lsp-installer#480]
### Problem description

Failed to install powershell_es because the server requires TLS1.2. Is it my system's configuration or something that the plugin should handle?

### Neovim version (>= 0.6)

NVIM v0.6.1
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
Compiled by runneradmin@fv-az152-430

### Operating system/version

Windows Server 2016 Standard

### I've recently downloaded the latest plugin version of both nvim-lsp-installer and nvim-lspconfig

- [X] Yes

### Affected language servers

powershell_es

### Actual behavior

The plugin fails to install the language server powershell_es

### Expected behavior

Install successfully

### LspInstallInfo output

```shell
◍ powershell_es (failed)
        Fetching latest release version from GitHub API...
        iwr : The request was aborted: Could not create SSL/TLS secure channel.
        At line:1 char:42
        + ... lyContinue';iwr -Headers @{'User-Agent' = 'nvim-lsp-installer (+https ...
        +                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
            + CategoryInfo          : InvalidOperation: (System.Net.HttpWebRequest:HttpWebRequest) [Invoke-WebRequest], WebExc 
           eption
            + FullyQualifiedErrorId : WebCmdletWebResponseException,Microsoft.PowerShell.Commands.InvokeWebRequestCommand
```
```


### Installation log

```shell
[INFO  2/14/2022 2:34:35 PM] ...-installer/lua/nvim-lsp-installer/ui/status-win/init.lua:644: Starting install server_name="powershell_es", requested_version=""
[ERROR 2/14/2022 2:34:35 PM] ...rt/nvim-lsp-installer/lua/nvim-lsp-installer/process.lua:153: Failed to spawn process. cmd="gh", err="ENOENT: no such file or directory"
[ERROR 2/14/2022 2:34:36 PM] ...rt/nvim-lsp-installer/lua/nvim-lsp-installer/process.lua:153: Failed to spawn process. cmd="wget", err="ENOENT: no such file or directory"
[INFO  2/14/2022 2:34:38 PM] ...-installer/lua/nvim-lsp-installer/ui/status-win/init.lua:663: Installation completed server_name="powershell_es", success=false
```

### Healthcheck

```shell
## nvim-lsp-installer report
  - OK: neovim version >= 0.6.0
  - WARNING: **Go**: not available
  - WARNING: **Ruby**: not available
  - WARNING: **RubyGem**: not available
  - WARNING: **Composer**: not available
  - WARNING: **PHP**: not available
  - ERROR: **npm**: not available
  - ERROR: **node**: not available
  - WARNING: **python**: not available
  - WARNING: **pip**: not available
  - WARNING: **python3**: not available
  - WARNING: **pip3**: not available
  - WARNING: **javac**: not available
  - WARNING: **julia**: not available
  - ERROR: **wget**: not available
  - ERROR: **gzip**: not available
  - ERROR: **tar**: not available
  - OK: **curl**: `curl 7.77.0 (x86_64-pc-win32) libcurl/7.77.0 OpenSSL/1.1.1k (Schannel) zlib/1.2.11 brotli/1.0.9 zstd/1.5.0 libidn2/2.3.1 libssh2/1.9.0 nghttp2/1.43.0 libgsasl/1.10.0
`
  - OK: **java**: `Ok`
```

**Comment by williamboman** [comment]
Hello! Ah hmmm Windows issues.. Are you perhaps behind some sort of proxy?

**Comment by williamboman** [comment]
So the API that's causing this is GitHub's, which started refusing `TLSv1/TLSv1.1` [quite some time ago](https://github.blog/2018-02-01-crypto-removal-notice/) (2018). Let me do some research on `iwr`..

**Comment by williamboman** [comment]
Hey @sakhnik, could you try the branch `fix-iwr-tls` (#487) and see if it fixes things?

**Comment by sakhnik** [comment]
Sure it does, thanks!
